elated-egrets/bangazon-internal-site
# pzc training back button

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git checkout pzc-training-back-button
```

## Steps to test

1. Start virtual environment
1. Nuke DB/Migrations
1. Run:
    - ```
       python manage.py makemigrations employee_portal
       python manage.py migrate
       django_data.sh employee_portal faker_factory
       ```
1.Make sure Django server is running
1. run 
    - `python manage.py runserver`
1. click into training
1. click into a single training detail page
1. see back button
1. click back button to go back to list

## What you changed
Training program detail page


## What was added to read me

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Has a teammate approved the PR?

Then Ship It!
